### ISSUE TITLE
Make readPlaylist method usable from other JVM languages

### ISSUE THREAD

**Issue by laurio** [carlanton/m3u8-parser#13]
Currently it's not possible to call the `readPlaylist` method from Clojure and possibly from other JVM languages because the `AbstractPlaylistParser` class is package private.
Would it be possible to make it public?

**Comment by carlanton** [comment]
Hi!

Why do you want to use the AbstractPlaylistParser directly? Have you tried
to use MasterPlaylistParser and MediaPlaylistParser?

**Comment by laurio** [comment]
Hi!

Actually I'm just trying to use `MasterPlaylistParser` as described at https://github.com/carlanton/m3u8-parser#parse-master-playlist - executing the following code
```clojure
(let [player (MasterPlaylistParser.)]
    (.readPlaylist
      player
      (Paths/get (URI. "file:///./manifest.m3u8"))))
```
throws:
```
Execution error (IllegalArgumentException) at clj-carlanton-m3u8-parser-test.minimal-repro/eval1577 (minimal_repro.clj:2).
No matching method readPlaylist found taking 1 args for class io.lindstrom.m3u8.parser.MasterPlaylistParser
```
For this specific case, making AbstractPlaylistParser `public` will fix the
problem. This should have no impact on other code. The class will
still not be instantiable because it is also `abstract`.

Please see more details about this issue at https://github.com/netty/netty/issues/1780 and https://stackoverflow.com/a/36999820/9615

**Comment by carlanton** [comment]
Well that's a weird bug! Fixed in 0.11 which should be on maven in ~1 hour or so.

Thanks,
Anton

**Comment by laurio** [comment]
Thank you!
